editor/plugins/org.fusesource.ide.projecttemplates/META-INF/MANIFEST.MF

```diff
@@ -55,13 +55,19 @@ Bundle-Activator: org.fusesource.ide.projecttemplates.internal.ProjectTemplatesA
 Export-Package: org.fusesource.ide.projecttemplates.actions;x-friends:="org.fusesource.ide.projecttemplates.tests.integration",
  org.fusesource.ide.projecttemplates.actions.ui;x-friends:="org.fusesource.ide.projecttemplates.tests.integration",
  org.fusesource.ide.projecttemplates.adopters,
+ org.fusesource.ide.projecttemplates.adopters.configurators,
+ org.fusesource.ide.projecttemplates.adopters.creators,
  org.fusesource.ide.projecttemplates.adopters.util,
  org.fusesource.ide.projecttemplates.impl.medium,
  org.fusesource.ide.projecttemplates.impl.simple,
  org.fusesource.ide.projecttemplates.internal;x-friends:="org.fusesource.ide.projecttemplates.tests.integration",
  org.fusesource.ide.projecttemplates.maven,
- org.fusesource.ide.projecttemplates.util;x-friends:="org.fusesource.ide.projecttemplates.tests.integration,org.fusesource.ide.camel.validation.tests.integration,org.fusesource.ide.launcher.tests.integration",
- org.fusesource.ide.projecttemplates.util.camel;x-friends:="org.fusesource.ide.projecttemplates.tests.integration",
- org.fusesource.ide.projecttemplates.wizards;x-friends:="org.fusesource.ide.projecttemplates.tests.integration",
+ org.fusesource.ide.projecttemplates.util;
+  x-friends:="org.fusesource.ide.projecttemplates.tests.integration,
+   org.fusesource.ide.camel.validation.tests.integration,
+   org.fusesource.ide.launcher.tests.integration,
+   org.fusesource.ide.syndesis.extension.ui",
```

Comment:
when starting to have non-test plugin in x-friends I think that the export package needs to be considered as part of the API and remove the x-friends